core/start-service.sh

```diff
@@ -115,7 +117,7 @@ search)
 spooler)
 	dockerize \
 		-wait file://var/run/kopano/server.sock \
-		-wait tcp://mail:25 \
+		-wait tcp://$KCCONF_SPOOLER_SMTP_SERVER:25 \
```

Comment:
Actually one more thing. From the looks of this, this either needs to have a default value or needs to be added to docker-compose.yml (with a default value matching the old case) , else startup will fail for everyone after merge.

Reply:
Wait nevermind. this is already in compose.